rlberry/manager/agent_manager.py

```diff
@@ -729,8 +732,20 @@ def load(cls, filename):
 
         obj.__dict__.clear()
         obj.__dict__.update(tmp_dict)
+
         return obj
 
+    def __eq__(self, other):
+        self_agent_init_kwargs = deepcopy(self.init_kwargs)[0]
```

Comment:
I think we should also check the equality of `eval_kwargs` and `fit_kwargs`

Reply:
The advantage of my hash approach is that I tested everything (i.e. even new attributes that the user may have defined in AgentManager, it also tested eval env, train env... everything). But maybe it is not really necessary.

Reply:
If needed, I think the hash approach can also be done in `__eq__`, no? Something like `hash(self) == hash(other)`

Reply:
I think let us go for the simple solution. If people really want to be reproducible they would have to use container or something like that and not only this kind of software level check. I will keep it simple and only test the `{eval,fit,init}_kwargs` and the agent_class this should be sufficient.

Reply:
Done.